reference/README.md

```diff
@@ -112,7 +112,7 @@ If `propagate` is set to `False`, no flags will be passed to `make`.
 | `reference` | The set of reference values for this test. | `{}` | scoped dictionary | 2.2 |
 | `sanity_patterns` | The set of sanity patterns to look for for this test. | `None` | See [Output parsing](#output-parsing-and-performance-assessment) | 2.2 |
 | `sourcepath` | The path to the source file or source directory of the test. If not absolute, it is resolved against the `sourcesdir` directory. If it refers to a regular file, this file will be compiled (its language will be automatically recognized) and the produced executable will be placed in the test's stage directory. If it refers to a directory, this will be copied to the test's stage directory and `make` will be invoked in that. | `''` | string | 2.2 |
-| `sourcesdir` | The directory containing the test's resources. | `<test-prefix>/src/` | string | 2.2 |
+| `sourcesdir` | The directory containing the test's resources. In a `RunOnlyRegressionTest`, you can set this field to `None` if you do not need any resources at all. | `<test-prefix>/src/` | string | 2.2 |
```

Comment:
You should also add a note in the "NOTES" section later that the behaviour has changed from version 2.5 onward.

Reply:
I added a note.